fvtest/porttest/omrsockTest.cpp

```diff
@@ -509,6 +515,41 @@ TEST(PortSockTest, two_socket_stream_communication)
 	omrsock_socket_t connectedClientStreamSocket = NULL;
 	EXPECT_EQ(OMRPORTLIB->sock_accept(OMRPORTLIB, serverStreamSocket, &connectedClientStreamSockAddr, &connectedClientStreamSocket), 0);
 
+	char msg[100] = "This is an omrsock test for 2 socket stream communications.";
+	int32_t bytesLeft = strlen(msg) + 1;
+	uint8_t *cursor = (uint8_t *)msg;
+	int32_t bytesSent = 0;
+	while(1) {
+		bytesSent = OMRPORTLIB->sock_send(OMRPORTLIB, connectedClientStreamSocket, cursor, bytesLeft, 0);
+		if (bytesSent < 0) {
+			FAIL();
+			break;
+		}
+		bytesLeft -= bytesSent;
+		cursor += bytesSent;
+		if (0 == bytesLeft) {
+			break;
+		}
+	}
+
+	char buf[100] = "";
```

Comment:
I think it would be better to zero-initialize the entire array:
```suggestion
	char buf[100] = {0};
```